com.avaloq.tools.ddk.check.core/src/com/avaloq/tools/ddk/check/generator/CheckGenerator.xtend

```diff
@@ -45,9 +49,11 @@ class CheckGenerator extends JvmModelGenerator {
         CheckGeneratorConstants::CHECK_REGISTRY_OUTPUT,
         catalog.generateServiceRegistry(CheckUtil::serviceRegistryClassName, fsa)
       )
-
-      // change output path for html files to docs/
-      fsa.generateFile(catalog.docFileName, CheckGeneratorConstants::CHECK_DOC_OUTPUT, catalog.compileDoc)
+      // generate documentation for SCA-checks only
+	  if(!config.isGenerateLanguageInternalChecks){

```

Comment:
Wonky indentation caused by stray tab characters, and a missing space on either side of the parentheses. (Important stuff I know!)